common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsAuthorizationRequest.java

```diff
@@ -54,8 +55,8 @@
     private static final String CORRELATION_ID = "client-request-id";
     private static final String LOGIN_REQ = "login_req";
     private static final String DOMAIN_REQ = "domain_req";
-    private static final String SCOPE_PROFILE = "profile";
-    private static final String[] RESERVED_SCOPES = {"openid", SCOPE_PROFILE, "offline_access"};
+    //TODO: Should this be in the request or in the oAuth strategy?
```

Comment:
Is there any reason why we want to put it into oauth strategy? It is easier to understand if we keep the reserved scopes inside the request. Because RESERVED_SCOPES is part of scope. And scope belongs to authorization request.

Reply:
Maybe... i'd like to ponder this a bit.  Trying to keep plain old java objects used for serialization as simple as possible.

Reply:
To keep AuthorizationRequest "plain", do you also suggest we should move the authorization url constructor to somewhere else?

Reply:
Yes.  I think we should do that.  We can continue to make changes as we go though